src/components/application_manager/test/resumption/resumption_data_test.cc

```diff
@@ -490,6 +490,7 @@ void ResumptionDataTest::SetCommands() {
   sm::SmartObject sm_icon;
   for (uint32_t i = 0; i < kCountOfCommands_; ++i) {
     char numb[12];
+    // cppcheck-suppress invalidPrintfArgType_sint
```

Comment:
@Ypostolov Do not suppress this issue. You can find solution at several comment above.